### PR TITLE
More db schemas

### PR DESCRIPTION
**Pull request by Giems**

### CODE REVIEW

File: database/migrations/000000000004_request_status.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CREATE TYPE request_status_enum AS ENUM (
+    'Pending', 
+    'Completed', 
+    'Failed', 
+    'Rejected', 
+    'TimedOut', 
+    'Unknown'
+);
```
Comment (NorbertBodziony) [inline database/migrations/000000000004_request_status.sql:8]: what does failed and unknown mean?

File: database/migrations/000000000003_sessions.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    session_id TEXT NOT NULL UNIQUE,
+    app_id TEXT NOT NULL,
+    app_metadata TEXT NOT NULL,
+    app_connection_address TEXT NOT NULL,
+    persistent BOOLEAN NOT NULL,
+    network TEXT NOT NULL,
+    client_id TEXT,
+    client_device TEXT,
+    client_metadata TEXT,
+    client_notification_endpoint TEXT,
```
Comment (NorbertBodziony) [inline database/migrations/000000000003_sessions.sql:11]: this can be null ?

**Review by NorbertBodziony — CHANGES_REQUESTED** [review]

File: database/migrations/000000000004_request_status.sql (hunk 1 above)
Comment (Giems) [reply database/migrations/000000000004_request_status.sql:8]: removed

File: database/src/structs/subscription.rs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+use sqlx::Type;
+
+#[derive(Clone, Debug, Eq, PartialEq, Type)]
+#[sqlx(type_name = "subscription")]
+pub struct Subscription {
+    pub email: String,
+    pub subscribed_at: i64,
+}
```
[the remark this answers is not in the document]
Comment (Giems) [reply database/src/structs/subscription.rs:8]: updated type based on sql definition

**Pull request merged**